frontend/src/app/main/data/workspace/colors.cljs

```diff
@@ -82,7 +82,7 @@
           (assoc-in [:workspace-global :picked-shift?] shift?)))))
 
 (defn transform-fill
-  [state ids color transform]
+  [state ids color transform & options]
```

Comment:
I'm not clearly understand this change, I gues it should be `& {:as options}`, but as it is used around the changed code it always implies using `apply` that not looks good.

Reply:
Changed. Used arities option because some involved functions are in the critical path, I think. And also the `update-shapes` uses it.